docs/nextjs.md

```diff
@@ -5,6 +5,105 @@ Mongoose works out of the box with Next.js.
 If you're looking to get started, please use [Next.js' official Mongoose sample app](https://github.com/vercel/next.js/tree/canary/examples/with-mongodb-mongoose).
 Furthermore, if you are using Next.js with [Vercel Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions), please review [Mongoose's AWS Lambda docs](https://mongoosejs.com/docs/lambda.html).
 
+## Quick Start
+
+Here's a basic example of using Mongoose with Next.js App Router:
+
+```javascript
+// lib/mongodb.js
+import mongoose from 'mongoose';
+
+const MONGODB_URI = process.env.MONGODB_URI;
+
+if (!MONGODB_URI) {
+  throw new Error('Please define the MONGODB_URI environment variable');
+}
+
+let cached = global.mongoose;
+
+if (!cached) {
+  cached = global.mongoose = { conn: null, promise: null };
+}
+
+async function dbConnect() {
```

Comment:
Overall this entire function could be drastically simplified by having something like `if (!cached) { cached = mongoose; await mongoose.connect(); }`